### ISSUE TITLE
is L-GM-loss updated with keras code?

### ISSUE THREAD

**Issue by clscy** [WeitaoVan/L-GM-loss#5]
Hi, thank you for this wonderful work. 
And I wonder that whether you will release the keras version of L-GM loss.
Look forward to your reply.

**Comment by alyato** [comment]
@clscy  is there the version of keras. thanks.

**Comment by WeitaoVan** [comment]
We've been working on it.
And the keras code will be released soon.
Thanks for asking.
